code/modules/projectiles/guns/projectile/boltgun.dm

```diff
@@ -13,7 +13,7 @@
 	fire_delay = 2
 	damage_multiplier = 1.1
 	penetration_multiplier  = 1.5
-	recoil_buildup = 40 //same as AMR
+	recoil_buildup = 10 //FORMERLY same as AMR, whos idea was this?
```

Comment:
WHAT THE HELL IT HAD THAT MUCH?

Honestly though, even if it shouldn't, I'm a bit afraid this might make the rifle way too good for troopers as they have the ability to shoot bolt action rifles as if they were semiauto ones, making every shot have a hell of a lot less recoil, effectively turning a Mosin/Kardeshev into an M14

Then again, like the box says, it IS rather looked down upon despite being a really damn good rifle for its caliber, multipliers and stats.

Reply:
This point was raised by rebel as well- might raise to fifteen or possibly twenty then we'll see from there how it actually feels in play

Reply:
20 Novakovic - 18 Nagant would be a good middle ground for testing purposes, although honestly kickback is **really** felt over 30, less than it on semi auto is barely noticeable with good enough stats